dolo/compiler/model_numeric.py

```diff
@@ -179,6 +179,14 @@ def _repr_html_(self):
             equations = {"dynare": self.symbolic.equations}
         else:
             equations = self.symbolic.equations
+            # Create definitions equations and append to equations dictionary
+            definitions = self.symbolic.definitions
+            tmp = []
+            for deftype in definitions:
+                tmp.append(deftype + '=' + definitions[deftype])
+            definitions = {'definitions': tmp}
+            equations.update(definitions)
```

Comment:
Ideally, no. But this was a straightforward way to have the definitions pulled into the same table as the equations.

@albop suggested printing the definitions (and also model symbols) in a separate table, in which case equations wouldn't need to be appended with definitions. But that's not urgent, and also probably best left for someone who knows their way around html better than I do.